### DISCUSSION TITLE
Macro to get the current row, column, and total number of columns of a tblr?

### DISCUSSION THREAD

**Discussion by kalekje** [lvjr/tabularray#495] [Q&A]
I'm working towards building some wrapper macros for this package. I would like my macros to be "aware" of the current column and row number of the tblr. For example:

```
\begin{tblr}{lll}
  \tblrCurrentRow = 1 &                      & \tblrCurrentCol = 3  \\
  \tblrCurrentRow = 2 & \tblrCurrentCol = 2  &  \\
\end{tblr}
```

Do these exist?

**Comment by muzimuzhi** [comment]
Counters `rownum` and `colnum` hold the current row and column number, and `rowcount` and `colcount` hold the total number of rows and columns. For example `\arabic{rownum}` gives the current row.

See package manual (v2024A, 2024-02-16) sec. 3.8 "Counters and Lengths" for more info.
